src/adafruit_blinka/microcontroller/mcp2221/mcp2221.py

```diff
@@ -285,6 +285,8 @@ def _i2c_read(self, cmd, address, buffer, start=0, end=None):
                     continue
                 if resp[2] in (RESP_READ_COMPL, RESP_READ_PARTIAL):
                     break
+            else:
+                raise RuntimeError("I2C write error: max retries reached.")
```

Comment:
shouldn't this say "I2C read error;"

Reply:
Yep, I wanted to make sure I go the wording exactly like the write version. I'll update it.